apps/webpack.js

```diff
@@ -85,8 +85,14 @@ if (envConstants.COVERAGE) {
   ];
 }
 
+var devtool = function () {
+  return process.env.CHEAP ?
+    'cheap-inline-source-map' :
+    'inline-source-map';
+};
```

Comment:
does this need to be a function vs. just `var devtool = process.env.CHEAP ? 'cheap-inline-source-map' : 'inline-source-map';`